docs/content/tutorials/hello-monkey.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Tutorial 1: Hello Monkey"
+date: 2020-05-26T20:57:10+03:00
+draft: false
+pre: '<i class="fab fa-graduation-cap"></i> '
+weight: 2
+tags: ["tutorials", "hello-monkey"]
+---
+
+## Hello Monkey
+
+In this tutorial, we will use the Infection Monkey to exploit a simple vulnerability. We will configure Infection Monkey to exploit the vulnerability, and then run the Monkey to observe that it is able to gain access to the machine. You'll learn how to start Infection Monkey, configure it, and run the Monkey against a network. Specifically, you'll learn how to:
+- Install plugins
+- Specify targets for Infection Monkey to exploit in the network
+- Tell Infection Monkey what exploits to attempt against machines on the network
+- Provide Infection Monkey with credentials it can use when attempting to exploit machines
+- Run the Monkey and observe its progress
+
+### Prerequisites
+First, make sure that you have the following prerequisites installed:
+- `docker` and `docker-compose`
+
+### Run the environment
+Next, we'll use docker compose to run Infection Monkey along with our vulnerable container.
+
+Run `docker compose up` to start the environment
+
+{{% notice note %}}
+The current docker compose script uses the host network.
+{{% /notice %}}
+
+Now that the environment is running, open a browser to `https://localhost:5000` to access the Monkey Island web interface. You will be presented with a registration page if you haven't registered. Provide a username and password to log in.
+
+TODO: Screenshot of monkey login screen
+
+
+### Configure the Monkey
+Now we'll need to configure Infection Monkey to exploit the vulnerable container. Select "Configure Monkey" on the "Getting Started" page (or select _Configuration_ in the navigation sidebar).
+
+Before the Monkey can do anything useful, it needs to be configured. Otherwise, it won't know what exploits to attempt, or what machine(s) to attempt to breach.
+
+{{% notice note %}}
+The Infection Monkey will only attempt to breach the machines that you've explicitly configured it to target. This helps to ensure that the Monkey doesn't run amok on your network.
```

Comment:
What is "amok"?

Reply:
![image](https://github.com/guardicore/monkey/assets/19957806/75dce9ee-b44a-42a6-8c57-3dfdd11411db)
https://www.merriam-webster.com/dictionary/amok